internal/v1/handler.go

```diff
@@ -455,7 +467,7 @@ func (h *Handlers) GetComposeMetadata(ctx echo.Context, composeId uuid.UUID) err
 		}
 		return echo.NewHTTPError(http.StatusNotFound, string(body))
 	} else if resp.StatusCode != http.StatusOK {
-		httpError := echo.NewHTTPError(http.StatusInternalServerError, "Failed querying compose status")
+		httpError := echo.NewHTTPError(http.StatusInternalServerError, fmt.Sprintf("Failed querying compose status"))
```

Comment:
AFAICT the Sprintf is not needed here. Maybe also switch to "Failed querying compose metadata" or similar (as the call that fails above is "h.server.cClient.ComposeMetadata(composeId)"?

Reply:
I missed the removing of 'Sprintf' word, sorry about that